sentinelsat/sentinel.py

```diff
@@ -1438,3 +1438,40 @@ def _parse_odata_response(product):
                 pass
         output[attr["Name"]] = value
     return output
+
+
+def placename_to_wkt(placename):
+    """Geocodes the placename to rectangular bounding extents using Nominatim API and
+       returns the corresponding 'ENVELOPE' form Well-Known-Text.
+
+    Parameters
+    ----------
+    placename : str
+        the query to geocode
+
+    Returns
+    -------
+    full name and coordinates of the queried placename
+        list in the form [string placeinfo, wkt string in form 'ENVELOPE(minX, maxX, maxY, minY)']
```

Comment:
The order of return values in docs does not match the actual order.

Reply:
woops haha